search_service/proxy/elasticsearch.py

```diff
@@ -116,6 +116,8 @@ def _get_search_result(self, page_index: int,
 
         for hit in response:
             try:
+                es_metadata = hit.__dict__.get('meta', {})
```

Comment:
it has been a while, could you add an example on the return value of `hit.__dict__` for docstring ? is it getting the existing ES document?

Reply:
sure, will do.
yes, it's getting the existing ES documnet.